### PR TITLE
Refactor module to add composer/whmcs meta and updated structure

### PR DESCRIPTION
**Pull request by meramsey**
This will need to be merged after the https://github.com/monero-integrations/monerowhmcs/pull/41 is merged as it has some of its fixes, but is minor when not factoring those in. Or alternatively if you want to merge this instead that works and the other PR for 41 can be closed out.

Basically this better restructures stuff to match WHMCS's recommended structure for these kinds of modules. 
https://github.com/WHMCS/sample-gateway-module
https://github.com/WHMCS/sample-addon-module

More about the whmcs.json:
https://developers.whmcs.com/advanced/json-file/

When in WHMCS looking through Modules to activate or view activated ones if the whmcs.json is missing it looks like the below
Screenshot from 2022-08-07 10-53-38

Clicking shows empty details.
Screenshot from 2022-08-07 10-57-31

With the whmcs.json it looks nice like the below and the others.
Screenshot from 2022-08-07 10-53-57
Screenshot from 2022-08-07 10-56-39

Addon doesn't do logos but does look nice
Screenshot from 2022-08-07 11-08-44

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "monero-integrations/monerowhmcs",
+  "description": "A WHMCS Payment Gateway for accepting Monero.",
+  "keywords": [
+    "whmcs",
+    "web host automation platform",
+    "third party gateway module",
+    "gateway module",
+    "third party gateway",
```
Comment (serhack) [inline composer.json:9]: It's not a common "third party gateway", I would say "self-managed payment gateway", or "payment gateway".

**Review by serhack — CHANGES_REQUESTED** [review]
Thank you a lot for the work you're doing. I appreciated the PR and issues you're creating.

Before reviewing, could you please update this PR with the actual branch?

**Comment by meramsey** [comment]
> Thank you a lot for the work you're doing. I appreciated the PR and issues you're creating.
> 
> Before reviewing, could you please update this PR with the actual branch?

Should be all caught up and I merged the stuff from https://github.com/monero-integrations/monerowhmcs/pull/44 into this one as well as they would have had to be redone anyways after this merge due to the path changes for the addon module structure.

Should be all set :) @serhack

**Review by serhack — APPROVED** [review]
Thank you for the PR!

**Pull request merged**